jep/202/README.adoc

```diff
@@ -130,7 +128,7 @@ There are serious drawbacks to use filesystem when running in cloud or container
 * Causing all sorts of issues due to usage of remoting for file transfer.
 Going through the master as files are copied from/to the agent cause load, cpu, network issues that are hard to diagnose and recover from.
 * Causing a big performance hit due to the traffic and disk access, preventing the usage of distributed filesystems such as NFS, GlusterFS,...
-* Providing limited functionality compared to a first class blob store (AWS S3, Azure Blob Storage, Google Blobstore) or artifact repository (Nexus, Artifactory).
+* Providing limited functionality compared to a first class blob store (AWS S3, Azure Blob Storage, Google Blobstore).
```

Comment:
@jglick Sorry, but I'm afraid that's putting incorrect words in my mouth -- what I said is that they were "mentioned" originally in JEP-202 which is 100% true.  I never said you were advocating for it, but that you seem to be making a completely arbitrary distinction for what can and cannot integrate with the Jenkins artifact mechanism as an external store.   As far as I'm concerned the reason seems to be more implementation convenience than anything in the user experience.

I will remind you that you made similar arbitrary assumptions about what users would be doing with Pipeline files, and that they would be quite simple -- assumptions that have proven incorrect. 

To me, if feels obvious to ask "okay, so you support S3 now, when can I just use Artifactory/Nexus/whatever and drop the custom steps to upload artifacts?"  It lets people simplify their CI/CD cycle by making the actual artifact store a mere configuration detail.

Reply:
> the reason seems to be more implementation convenience than anything in the user experience

No, the reason is very much user experience. If someone does ask

> when can I just use Artifactory/Nexus/whatever and drop the custom steps to upload artifacts?

then the answer will be “never”. It would be a step backwards.